spec/session_helper.rb

```diff
@@ -0,0 +1,26 @@
+module SessionHelpers
+  def sign_up(email='joeb@test.com')
+    visit '/'
+    log_out
+    click_link 'Sign up'
+    fill_in 'user_email', with: email
+    fill_in 'user_password', with: 'password'
+    fill_in 'user_password_confirmation', with: 'password'
+    click_button 'Sign up'
+  end
+
+  def log_in(user)
+    visit '/'
+    log_out
+    click_link 'Log in'
+    fill_in 'user_email', with: user.email
+    fill_in 'user_password', with: 'password'
+    click_button 'Log in'
+  end
+
+  def log_out
+    if page.has_link? 'Log out'
```

Comment:
Style/GuardClause: Use a guard clause instead of wrapping the code inside a conditional expression. (https://github.com/bbatsov/ruby-style-guide#no-nested-conditionals)<br>Style/IfUnlessModifier: Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`. (https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier)